.github/ISSUE_TEMPLATE/B-submit-a-presubmission-inquiry.md

```diff
@@ -5,11 +5,16 @@ about: Not sure your software fits, or submitting a statistical package? Use thi
 
 ---
 
-
-Submitting Author: Name (@github_handle)  
-Other Package Authors: (delete if none) Name (@github_handle)
-Repository:  <!--repourl--><!--end-repourl-->
+<!---
+Below, please enter values for (1) submitting author GitHub handle (replacing "@github_handle@); and (2) Repository URL (replacing "https://repourl"). Values for additional package authors may also be specified, replacing "@github_handle1", "@github_handle2" - delete these if not needed. DO NOT DELETE HTML SYMBOLS (everything between "<!" and ">"). Replace only "@github_handle" and "https://repourl". This comment may be deleted once it has been read and understood.
```

Comment:
```suggestion
Below, please enter values for (1) submitting author GitHub handle (replacing "@github_handle"); and (2) Repository URL (replacing "https://repourl"). Values for additional package authors may also be specified, replacing "@github_handle1", "@github_handle2" - delete these if not needed. DO NOT DELETE HTML SYMBOLS (everything between "<!" and ">"). Replace only "@github_handle" and "https://repourl". This comment may be deleted once it has been read and understood.
```

Reply:
Maybe add why not to remove the HTML comments ("as these are needed by our automatic tooling to check submissions" or so)

Reply:
I agree but suspect it might be worth waiting first to gauge general responses to current versions.